controllers/apply/form-router-next/index.js

```diff
@@ -238,6 +238,14 @@ function initFormRouter({
      * Route: Edit application
      */
     router.get('/edit/:applicationId', function(req, res) {
+        // If this link includes a source (s) parameter, track it
+        // eg. to analyse usage of expiry reminder emails
+        if (get(req.query, 's') === 'expiryEmail') {
```

Comment:
I think `if(req.query.s === 'expiryEmail')` should be safe to do without the `get`